.expo/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "hostType": "tunnel",
+  "lanType": "ip",
+  "dev": true,
+  "strict": false,
+  "minify": false,
+  "urlType": "exp",
+  "urlRandomness": null
+}
```

Comment:
@JulianKingman is this file needed?

Reply:
@helielson that's an expo file, which should be ignored. I updated the gitignore (expo directory was the outdated `.exponent`, instead of `.expo`), and removed it.